docs/RELEASE_NOTES.md

```diff
@@ -1,11 +1,18 @@
 # Release Notes
 ## v6.0
 _May 2020:_ Version 6.0
-**Note:** This is a major release and therefore requires a schema change. See the deploy steps section for details.
+**Note:** This is a major release and therefore requires a schema change. See the deploy steps section for details. Anything marked as *experimental* has not yet been tested on a production texting campaign, but has been QA'd by our QA volunteers.
```

Comment:
I'm not sure we should suggest even that -- our QA folks didn't test even environment with the phone number management enabled yet, much lest testing the feature.